client/scripts/views/modals/new_topic_modal.ts

```diff
@@ -72,23 +106,24 @@ const NewTopicModal: m.Component<{
               const disallowedCharMatches = text.match(/["<>%{}|\\/^`]/g);
               if (disallowedCharMatches) {
                 return [
-                  TextInputStatus.Error,
-                  `The ${pluralizeWithoutNumberPrefix(disallowedCharMatches.length, 'char')} 
-                  ${disallowedCharMatches.join(', ')} are not permitted`
+                  ValidationStatus.Failure,
+                  `The ${pluralizeWithoutNumberPrefix(
+                    disallowedCharMatches.length,
+                    'char'
+                  )} 
+                  ${disallowedCharMatches.join(', ')} are not permitted`,
                 ];
               } else {
-                return [TextInputStatus.Validate, 'Valid topic name'];
+                return [ValidationStatus.Success, 'Valid topic name'];
               }
             },
-            otherAttrs: {
-              autocomplete: 'off',
-              tabindex: 1,
-              oncreate: (vvnode) => {
-                // use oncreate to focus because autofocus: true fails when component is recycled in a modal
-                setTimeout(() => $(vvnode.dom).find('input').focus(), 0);
-              },
-              style: 'margin-bottom: 10px;'
-            }
+            autocomplete: 'off',
+            autofocus: true,
+            tabindex: 1,
+            oncreate: (vvnode) => {
+              // use oncreate to focus because autofocus: true fails when component is recycled in a modal
+              setTimeout(() => $(vvnode.dom).find('input').focus(), 0);
+            },
           }),
           m(FormGroup, [
             m(FormLabel, { for: 'description' }, 'Description'),
```

Comment:
Why not implement CWInput for all the inputs in this modal?

Reply:
I assumed the scope of the PR was restricted to existing v2 work. Make sense to leave further conversion until the design system is locked down?